### DISCUSSION TITLE
Clarify support for custom url schemes

### DISCUSSION THREAD

**Discussion by dmytrowix** [qutebrowser/qutebrowser#6052] [Q&A] [answered]
Hi,

I'm trying to make slack links works through qutebrowser. Slack links usually look like this
```
https://myorg.slack.com/archives/***
```

Upon opening in the browser, the page issues a request to a url with `slack:` scheme
```
slack://channel?team=***&id=***&message=***&thread_ts=***
```

The request however appears _canceled_ in devtools.

![image](https://user-images.githubusercontent.com/77284838/105338006-b65a3100-5be3-11eb-9e5f-e3c15e269ab1.png)

Is there a way to make qutebrowser handle specific schemes (delegating to `xdg-open` would do just fine)?

Just in case, not sure if related, I tried setting `unknwon_url_scheme_policy = allow-all` but it didn't help.

Thanks!

**Comment by toofar** [comment, marked as answer]
AFAIK custom protocol requests always show as cancelled in the devtools, even when there were successfully handled (not that the browser would know). I can't check right now but I'm pretty sure I got this working fine by setting content.unknwon_url_scheme_policy and registering a .desktop file with xdg-mine (apparently you can get it to work with a mimeapps or gconf too).
So I think if you make `xdg-open slack:...` work then clicking on those links in qutebrowser should work too (ad blocker allowing).
